examples/jsm/animation/MMDPhysics.js

```diff
@@ -1367,25 +1367,6 @@ class MMDPhysicsHelper extends Object3D {
 
 		}
 
-		function createCapsuleGeometry( radius, cylinderHeight, segmentsRadius, segmentsHeight ) {
-
-			var geometry = new CylinderGeometry( radius, radius, cylinderHeight, segmentsRadius, segmentsHeight, true );
-			var upperSphere = new Mesh( new SphereGeometry( radius, segmentsRadius, segmentsHeight, 0, Math.PI * 2, 0, Math.PI / 2 ) );
-			var lowerSphere = new Mesh( new SphereGeometry( radius, segmentsRadius, segmentsHeight, 0, Math.PI * 2, Math.PI / 2, Math.PI / 2 ) );
-
-			upperSphere.position.set( 0, cylinderHeight / 2, 0 );
-			lowerSphere.position.set( 0, - cylinderHeight / 2, 0 );
-
-			upperSphere.updateMatrix();
-			lowerSphere.updateMatrix();
-
-			geometry.merge( upperSphere.geometry, upperSphere.matrix );
```

Comment:
This code was actually broken since a matrix is no valid argument. It has to be an integer. This bug broke the merge and thus the entire geometry generation.

I guess this issue was overlooked when `Geometry` was deprecated since `BufferGeometry.merge()` works differently than `Geometry.merge()`.

Related #24443.